### PR TITLE
[round](decimalv2) round decimalv2 to precision value

### PR DESCRIPTION
**Pull request by Gabriel39**
## Proposed changes

before:
image
after :
image

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: gensrc/script/doris_builtins_functions.py [hunk 1]
```diff
@@ -1255,20 +1259,25 @@
...
         [['floor', 'dfloor'], 'DECIMAL32', ['DECIMAL32', 'INT'], ''],
         [['floor', 'dfloor'], 'DECIMAL64', ['DECIMAL64', 'INT'], ''],
         [['floor', 'dfloor'], 'DECIMAL128', ['DECIMAL128', 'INT'], ''],
+        [['ceil', 'dceil'], 'DECIMALV2', ['DECIMALV2', 'INT'], ''],
         [['ceil', 'dceil'], 'DECIMAL32', ['DECIMAL32', 'INT'], ''],
         [['ceil', 'dceil'], 'DECIMAL64', ['DECIMAL64', 'INT'], ''],
         [['ceil', 'dceil'], 'DECIMAL128', ['DECIMAL128', 'INT'], ''],
         [['truncate'], 'DOUBLE', ['DOUBLE', 'INT'], ''],
+        [['truncate'], 'DECIMALV2', ['DECIMALV2', 'INT'], ''],
         [['truncate'], 'DECIMAL32', ['DECIMAL32', 'INT'], ''],
```
Comment (yiguolei) [inline gensrc/script/doris_builtins_functions.py:1281]: add  some regression test please.

**Comment by Gabriel39** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by zclllyybb — APPROVED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**